lib/cc/cli/analyze.rb

```diff
@@ -48,6 +50,8 @@ def process_args
             @engine_options << @args.shift
           when "--dev"
             @dev_mode = true
+          when "-p", "--partial"
```

Comment:
This branch bumps complexity for this method over the threshold. While it's true I don't see how to make it simpler and keep readability.